packages/payload/src/fields/baseFields/baseLocalizedMeta.ts

```diff
@@ -0,0 +1,56 @@
+import type { Config, SanitizedConfig } from '../../config/types.js'
+import type { Field, FlattenedField } from '../config/types.js'
+export const baseLocalizedMetaFields = (
+  config: Config | SanitizedConfig,
+  flatten?: boolean,
+): Field[] => {
+  if (!config.localization || !config.localization.locales) {
+    return []
+  }
+
+  const localizedMetaFields = config.localization.locales.map((locale) => {
+    const code = typeof locale === 'string' ? locale : locale.code
+
+    return {
+      name: code,
+      type: 'group',
+      admin: {
+        disableBulkEdit: true,
+        disableListColumn: true,
+        disableListFilter: true,
+        hidden: true,
+      },
+      fields: [
+        {
+          name: 'status',
+          type: 'select',
+          options: [
+            { label: ({ t }: any) => t('version:draft'), value: 'draft' },
+            { label: ({ t }: any) => t('version:published'), value: 'published' },
+          ],
+        },
+        {
+          name: 'updatedAt',
+          type: 'date',
+        },
+      ],
+    }
+  })
+
+  return [
+    {
+      name: 'localizedMeta',
+      type: 'group',
+      admin: {
+        hidden: true,
+      },
+      fields: localizedMetaFields as Field[],
+      label: 'Localized Meta', // add translation
```

Comment:
Just flagging so its not missed. It's fine to do this last so we don't bloat the PR for now.